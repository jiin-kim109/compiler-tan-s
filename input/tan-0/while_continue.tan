main {
    var i := 2;
    while ( i <= 5 ) {
        if (i == 3) {
            i := i + 1 ;
            continue;
        }
        print i*i \s ;
        i := i + 1 ;
    }
    print \n ;
}