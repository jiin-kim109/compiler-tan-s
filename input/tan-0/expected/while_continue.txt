4 16 25
